new CI fails

### 🐛 Describe the bug

```
=========================== short test summary info ============================
FAILED test/datasets/test_elliptic.py::test_elliptic_bitcoin_dataset - TypeEr...
FAILED test/utils/test_convert.py::test_to_cugraph[False-True-edge_weight1]
FAILED test/utils/test_convert.py::test_to_cugraph[False-False-edge_weight1]
FAILED test/utils/test_convert.py::test_from_cugraph[True-True-None] - assert...
FAILED test/utils/test_convert.py::test_from_cugraph[True-True-edge_weight1]
FAILED test/utils/test_convert.py::test_from_cugraph[True-False-None] - asser...
FAILED test/utils/test_convert.py::test_from_cugraph[True-False-edge_weight1]
FAILED test/utils/test_convert.py::test_from_cugraph[False-True-None] - asser...
FAILED test/utils/test_convert.py::test_from_cugraph[False-True-edge_weight1]
FAILED test/utils/test_convert.py::test_from_cugraph[False-False-None] - asse...
FAILED test/utils/test_convert.py::test_from_cugraph[False-False-edge_weight1]
FAILED test/utils/test_spmm.py::test_spmm_basic[device1] - AssertionError: as...
==== 12 failed
```



```
=================================== FAILURES ===================================
________________________ test_elliptic_bitcoin_dataset _________________________
get_dataset = functools.partial(<function load_dataset at 0x7f5c16405ee0>, '/tmp/pyg_test_datasets')
    @onlyFullTest
    def test_elliptic_bitcoin_dataset(get_dataset):
>       dataset = get_dataset(name='EllipticBitcoinDataset')
test/datasets/test_elliptic.py:6: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 
test/conftest.py:41: in load_dataset
    return EllipticBitcoinDataset(path, *args, **kwargs)
/usr/local/lib/python3.8/dist-packages/torch_geometric-2.3.0-py3.8.egg/torch_geometric/datasets/elliptic.py:60: in __init__
    super().__init__(root, transform, pre_transform)
/usr/local/lib/python3.8/dist-packages/torch_geometric-2.3.0-py3.8.egg/torch_geometric/data/in_memory_dataset.py:57: in __init__
    super().__init__(root, transform, pre_transform, pre_filter, log)
/usr/local/lib/python3.8/dist-packages/torch_geometric-2.3.0-py3.8.egg/torch_geometric/data/dataset.py:97: in __init__
    self._process()
/usr/local/lib/python3.8/dist-packages/torch_geometric-2.3.0-py3.8.egg/torch_geometric/data/dataset.py:224: in _process
    self.process()
/usr/local/lib/python3.8/dist-packages/torch_geometric-2.3.0-py3.8.egg/torch_geometric/datasets/elliptic.py:90: in process
    x = torch.from_numpy(df_features.loc[:, 2:].values).to(torch.float)
/usr/local/lib/python3.8/dist-packages/pandas/core/indexing.py:873: in __getitem__
    return self._getitem_tuple(key)
/usr/local/lib/python3.8/dist-packages/pandas/core/indexing.py:1055: in _getitem_tuple
    return self._getitem_tuple_same_dim(tup)
/usr/local/lib/python3.8/dist-packages/pandas/core/indexing.py:750: in _getitem_tuple_same_dim
    retval = getattr(retval, self.name)._getitem_axis(key, axis=i)
/usr/local/lib/python3.8/dist-packages/pandas/core/indexing.py:1088: in _getitem_axis
    return self._get_slice_axis(key, axis=axis)
/usr/local/lib/python3.8/dist-packages/pandas/core/indexing.py:1122: in _get_slice_axis
    indexer = labels.slice_indexer(
/usr/local/lib/python3.8/dist-packages/pandas/core/indexes/base.py:4966: in slice_indexer
    start_slice, end_slice = self.slice_locs(start, end, step=step, kind=kind)
/usr/local/lib/python3.8/dist-packages/pandas/core/indexes/base.py:5169: in slice_locs
    start_slice = self.get_slice_bound(start, "left", kind)
/usr/local/lib/python3.8/dist-packages/pandas/core/indexes/base.py:5079: in get_slice_bound
    label = self._maybe_cast_slice_bound(label, side, kind)
/usr/local/lib/python3.8/dist-packages/pandas/core/indexes/base.py:5031: in _maybe_cast_slice_bound
    self._invalid_indexer("slice", label)
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 
self = Index([     'txId', 'time_step',           2,           3,           4,
                 5,           6,           7, ...        161,
               162,         163,         164,         165,         166],
      dtype='object', length=167)
form = 'slice', key = 2
    def _invalid_indexer(self, form: str_t, key):
        """
        Consistent invalid indexer message.
        """
>       raise TypeError(
            f"cannot do {form} indexing on {type(self).__name__} with these "
            f"indexers [{key}] of type {type(key).__name__}"
        )
E       TypeError: cannot do slice indexing on Index with these indexers [2] of type int
/usr/local/lib/python3.8/dist-packages/pandas/core/indexes/base.py:3267: TypeError
___________________ test_to_cugraph[False-True-edge_weight1] ___________________
edge_weight = tensor([0.4664, 0.3317, 0.4964, 0.4277]), directed = False
relabel_nodes = True
    @withPackage('cudf')
    @withPackage('cugraph')
    @pytest.mark.parametrize('edge_weight', [None, torch.rand(4)])
    @pytest.mark.parametrize('relabel_nodes', [True, False])
    @pytest.mark.parametrize('directed', [True, False])
    def test_to_cugraph(edge_weight, directed, relabel_nodes):
        import cugraph
    
        if directed:
            edge_index = torch.tensor([[0, 1, 1, 2], [1, 0, 2, 1]])
        else:
            edge_index = torch.tensor([[0, 1], [1, 2]])
    
        if edge_weight is not None:
            edge_weight[:edge_index.size(1)]
    
        graph = to_cugraph(edge_index, edge_weight, relabel_nodes, directed)
        assert isinstance(graph, cugraph.Graph)
        assert graph.number_of_nodes() == 3
    
        edge_list = graph.view_edge_list()
        assert edge_list is not None
    
        edge_list = edge_list.sort_values(by=['src', 'dst'])
    
        cu_edge_index = edge_list[['src', 'dst']].to_pandas().values
        assert edge_index.tolist() == cu_edge_index.T.tolist()
    
        if edge_weight is not None:
            cu_edge_weight = edge_list['weights'].to_pandas().values
>           assert edge_weight.tolist() == cu_edge_weight.tolist()
E           assert [0.4664377570...6652932167053] == [0.4664377570...7101001739502]
E             Left contains 2 more items, first extra item: 0.4963948[726](https://gitlab-master.nvidia.com/dl/dgx/pyg/-/jobs/53984430#L726)654053
E             Use -v to get the full diff
test/utils/test_convert.py:433: AssertionError
__________________ test_to_cugraph[False-False-edge_weight1] ___________________
edge_weight = tensor([0.4664, 0.3317, 0.4964, 0.4277]), directed = False
relabel_nodes = False
    @withPackage('cudf')
    @withPackage('cugraph')
    @pytest.mark.parametrize('edge_weight', [None, torch.rand(4)])
    @pytest.mark.parametrize('relabel_nodes', [True, False])
    @pytest.mark.parametrize('directed', [True, False])
    def test_to_cugraph(edge_weight, directed, relabel_nodes):
        import cugraph
    
        if directed:
            edge_index = torch.tensor([[0, 1, 1, 2], [1, 0, 2, 1]])
        else:
            edge_index = torch.tensor([[0, 1], [1, 2]])
    
        if edge_weight is not None:
            edge_weight[:edge_index.size(1)]
    
        graph = to_cugraph(edge_index, edge_weight, relabel_nodes, directed)
        assert isinstance(graph, cugraph.Graph)
        assert graph.number_of_nodes() == 3
    
        edge_list = graph.view_edge_list()
        assert edge_list is not None
    
        edge_list = edge_list.sort_values(by=['src', 'dst'])
    
        cu_edge_index = edge_list[['src', 'dst']].to_pandas().values
        assert edge_index.tolist() == cu_edge_index.T.tolist()
    
        if edge_weight is not None:
            cu_edge_weight = edge_list['weights'].to_pandas().values
>           assert edge_weight.tolist() == cu_edge_weight.tolist()
E           assert [0.4664377570...6652932167053] == [0.4664377570...7101001[739](https://gitlab-master.nvidia.com/dl/dgx/pyg/-/jobs/53984430#L739)502]
E             Left contains 2 more items, first extra item: 0.4963948726654053
E             Use -v to get the full diff
test/utils/test_convert.py:433: AssertionError
______________________ test_from_cugraph[True-True-None] _______________________
edge_weight = None, directed = True, relabel_nodes = True
    @withPackage('cudf')
    @withPackage('cugraph')
    @pytest.mark.parametrize('edge_weight', [None, torch.randn(4)])
    @pytest.mark.parametrize('directed', [True, False])
    @pytest.mark.parametrize('relabel_nodes', [True, False])
    def test_from_cugraph(edge_weight, directed, relabel_nodes):
        import cudf
        import cugraph
        from torch.utils.dlpack import to_dlpack
    
        if directed:
            edge_index = torch.tensor([[0, 1, 1, 2], [1, 0, 2, 1]])
        else:
            edge_index = torch.tensor([[0, 1], [1, 2]])
    
        if edge_weight is not None:
            edge_weight[:edge_index.size(1)]
    
        G = cugraph.Graph(directed=directed)
        df = cudf.from_dlpack(to_dlpack(edge_index.t()))
        if edge_weight is not None:
            df['2'] = cudf.from_dlpack(to_dlpack(edge_weight))
    
        G.from_cudf_edgelist(
            df,
            source=0,
            destination=1,
            edge_attr='2' if edge_weight is not None else None,
            renumber=relabel_nodes,
        )
    
        cu_edge_index, cu_edge_weight = from_cugraph(G)
    
>       assert cu_edge_index.tolist() == edge_index.tolist()
E       assert [[1, 0, 2, 1], [2, 1, 1, 0]] == [[0, 1, 1, 2], [1, 0, 2, 1]]
E         At index 0 diff: [1, 0, 2, 1] != [0, 1, 1, 2]
E         Use -v to get the full diff
test/utils/test_convert.py:469: AssertionError
__________________ test_from_cugraph[True-True-edge_weight1] ___________________
edge_weight = tensor([-0.2837,  0.9451, -0.3490,  0.4564]), directed = True
relabel_nodes = True
    @withPackage('cudf')
    @withPackage('cugraph')
    @pytest.mark.parametrize('edge_weight', [None, torch.randn(4)])
    @pytest.mark.parametrize('directed', [True, False])
    @pytest.mark.parametrize('relabel_nodes', [True, False])
    def test_from_cugraph(edge_weight, directed, relabel_nodes):
        import cudf
        import cugraph
        from torch.utils.dlpack import to_dlpack
    
        if directed:
            edge_index = torch.tensor([[0, 1, 1, 2], [1, 0, 2, 1]])
        else:
            edge_index = torch.tensor([[0, 1], [1, 2]])
    
        if edge_weight is not None:
            edge_weight[:edge_index.size(1)]
    
        G = cugraph.Graph(directed=directed)
        df = cudf.from_dlpack(to_dlpack(edge_index.t()))
        if edge_weight is not None:
            df['2'] = cudf.from_dlpack(to_dlpack(edge_weight))
    
        G.from_cudf_edgelist(
            df,
            source=0,
            destination=1,
            edge_attr='2' if edge_weight is not None else None,
            renumber=relabel_nodes,
        )
    
        cu_edge_index, cu_edge_weight = from_cugraph(G)
    
>       assert cu_edge_index.tolist() == edge_index.tolist()
E       assert [[1, 0, 2, 1], [2, 1, 1, 0]] == [[0, 1, 1, 2], [1, 0, 2, 1]]
E         At index 0 diff: [1, 0, 2, 1] != [0, 1, 1, 2]
E         Use -v to get the full diff
test/utils/test_convert.py:469: AssertionError
______________________ test_from_cugraph[True-False-None] ______________________
edge_weight = None, directed = False, relabel_nodes = True
    @withPackage('cudf')
    @withPackage('cugraph')
    @pytest.mark.parametrize('edge_weight', [None, torch.randn(4)])
    @pytest.mark.parametrize('directed', [True, False])
    @pytest.mark.parametrize('relabel_nodes', [True, False])
    def test_from_cugraph(edge_weight, directed, relabel_nodes):
        import cudf
        import cugraph
        from torch.utils.dlpack import to_dlpack
    
        if directed:
            edge_index = torch.tensor([[0, 1, 1, 2], [1, 0, 2, 1]])
        else:
            edge_index = torch.tensor([[0, 1], [1, 2]])
    
        if edge_weight is not None:
            edge_weight[:edge_index.size(1)]
    
        G = cugraph.Graph(directed=directed)
        df = cudf.from_dlpack(to_dlpack(edge_index.t()))
        if edge_weight is not None:
            df['2'] = cudf.from_dlpack(to_dlpack(edge_weight))
    
        G.from_cudf_edgelist(
            df,
            source=0,
            destination=1,
            edge_attr='2' if edge_weight is not None else None,
            renumber=relabel_nodes,
        )
    
        cu_edge_index, cu_edge_weight = from_cugraph(G)
    
>       assert cu_edge_index.tolist() == edge_index.tolist()
E       assert [[1, 0], [2, 1]] == [[0, 1], [1, 2]]
E         At index 0 diff: [1, 0] != [0, 1]
E         Use -v to get the full diff
test/utils/test_convert.py:469: AssertionError
__________________ test_from_cugraph[True-False-edge_weight1] __________________
edge_weight = tensor([-0.2837,  0.9451, -0.3490,  0.4564]), directed = False
relabel_nodes = True
    @withPackage('cudf')
    @withPackage('cugraph')
    @pytest.mark.parametrize('edge_weight', [None, torch.randn(4)])
    @pytest.mark.parametrize('directed', [True, False])
    @pytest.mark.parametrize('relabel_nodes', [True, False])
    def test_from_cugraph(edge_weight, directed, relabel_nodes):
        import cudf
        import cugraph
        from torch.utils.dlpack import to_dlpack
    
        if directed:
            edge_index = torch.tensor([[0, 1, 1, 2], [1, 0, 2, 1]])
        else:
            edge_index = torch.tensor([[0, 1], [1, 2]])
    
        if edge_weight is not None:
            edge_weight[:edge_index.size(1)]
    
        G = cugraph.Graph(directed=directed)
        df = cudf.from_dlpack(to_dlpack(edge_index.t()))
        if edge_weight is not None:
            df['2'] = cudf.from_dlpack(to_dlpack(edge_weight))
    
        G.from_cudf_edgelist(
            df,
            source=0,
            destination=1,
            edge_attr='2' if edge_weight is not None else None,
            renumber=relabel_nodes,
        )
    
        cu_edge_index, cu_edge_weight = from_cugraph(G)
    
>       assert cu_edge_index.tolist() == edge_index.tolist()
E       assert [[1, 0], [2, 1]] == [[0, 1], [1, 2]]
E         At index 0 diff: [1, 0] != [0, 1]
E         Use -v to get the full diff
test/utils/test_convert.py:469: AssertionError
______________________ test_from_cugraph[False-True-None] ______________________
edge_weight = None, directed = True, relabel_nodes = False
    @withPackage('cudf')
    @withPackage('cugraph')
    @pytest.mark.parametrize('edge_weight', [None, torch.randn(4)])
    @pytest.mark.parametrize('directed', [True, False])
    @pytest.mark.parametrize('relabel_nodes', [True, False])
    def test_from_cugraph(edge_weight, directed, relabel_nodes):
        import cudf
        import cugraph
        from torch.utils.dlpack import to_dlpack
    
        if directed:
            edge_index = torch.tensor([[0, 1, 1, 2], [1, 0, 2, 1]])
        else:
            edge_index = torch.tensor([[0, 1], [1, 2]])
    
        if edge_weight is not None:
            edge_weight[:edge_index.size(1)]
    
        G = cugraph.Graph(directed=directed)
        df = cudf.from_dlpack(to_dlpack(edge_index.t()))
        if edge_weight is not None:
            df['2'] = cudf.from_dlpack(to_dlpack(edge_weight))
    
        G.from_cudf_edgelist(
            df,
            source=0,
            destination=1,
            edge_attr='2' if edge_weight is not None else None,
            renumber=relabel_nodes,
        )
    
        cu_edge_index, cu_edge_weight = from_cugraph(G)
    
>       assert cu_edge_index.tolist() == edge_index.tolist()
E       assert [[1, 1, 2, 0], [0, 2, 1, 1]] == [[0, 1, 1, 2], [1, 0, 2, 1]]
E         At index 0 diff: [1, 1, 2, 0] != [0, 1, 1, 2]
E         Use -v to get the full diff
test/utils/test_convert.py:469: AssertionError
__________________ test_from_cugraph[False-True-edge_weight1] __________________
edge_weight = tensor([-0.2837,  0.9451, -0.3490,  0.4564]), directed = True
relabel_nodes = False
    @withPackage('cudf')
    @withPackage('cugraph')
    @pytest.mark.parametrize('edge_weight', [None, torch.randn(4)])
    @pytest.mark.parametrize('directed', [True, False])
    @pytest.mark.parametrize('relabel_nodes', [True, False])
    def test_from_cugraph(edge_weight, directed, relabel_nodes):
        import cudf
        import cugraph
        from torch.utils.dlpack import to_dlpack
    
        if directed:
            edge_index = torch.tensor([[0, 1, 1, 2], [1, 0, 2, 1]])
        else:
            edge_index = torch.tensor([[0, 1], [1, 2]])
    
        if edge_weight is not None:
            edge_weight[:edge_index.size(1)]
    
        G = cugraph.Graph(directed=directed)
        df = cudf.from_dlpack(to_dlpack(edge_index.t()))
        if edge_weight is not None:
            df['2'] = cudf.from_dlpack(to_dlpack(edge_weight))
    
        G.from_cudf_edgelist(
            df,
            source=0,
            destination=1,
            edge_attr='2' if edge_weight is not None else None,
            renumber=relabel_nodes,
        )
    
        cu_edge_index, cu_edge_weight = from_cugraph(G)
    
>       assert cu_edge_index.tolist() == edge_index.tolist()
E       assert [[1, 1, 2, 0], [0, 2, 1, 1]] == [[0, 1, 1, 2], [1, 0, 2, 1]]
E         At index 0 diff: [1, 1, 2, 0] != [0, 1, 1, 2]
E         Use -v to get the full diff
test/utils/test_convert.py:469: AssertionError
_____________________ test_from_cugraph[False-False-None] ______________________
edge_weight = None, directed = False, relabel_nodes = False
    @withPackage('cudf')
/usr/local/lib/python3.8/dist-packages/coverage/control.py:836: CoverageWarning: No data was collected. (no-data-collected)
  self._warn("No data was collected.", slug="no-data-collected")
    @withPackage('cugraph')
    @pytest.mark.parametrize('edge_weight', [None, torch.randn(4)])
    @pytest.mark.parametrize('directed', [True, False])
    @pytest.mark.parametrize('relabel_nodes', [True, False])
    def test_from_cugraph(edge_weight, directed, relabel_nodes):
        import cudf
        import cugraph
        from torch.utils.dlpack import to_dlpack
    
        if directed:
            edge_index = torch.tensor([[0, 1, 1, 2], [1, 0, 2, 1]])
        else:
            edge_index = torch.tensor([[0, 1], [1, 2]])
    
        if edge_weight is not None:
            edge_weight[:edge_index.size(1)]
    
        G = cugraph.Graph(directed=directed)
        df = cudf.from_dlpack(to_dlpack(edge_index.t()))
        if edge_weight is not None:
            df['2'] = cudf.from_dlpack(to_dlpack(edge_weight))
    
        G.from_cudf_edgelist(
            df,
            source=0,
            destination=1,
            edge_attr='2' if edge_weight is not None else None,
            renumber=relabel_nodes,
        )
    
        cu_edge_index, cu_edge_weight = from_cugraph(G)
    
>       assert cu_edge_index.tolist() == edge_index.tolist()
E       assert [[1, 0], [2, 1]] == [[0, 1], [1, 2]]
E         At index 0 diff: [1, 0] != [0, 1]
E         Use -v to get the full diff
test/utils/test_convert.py:469: AssertionError
_________________ test_from_cugraph[False-False-edge_weight1] __________________
edge_weight = tensor([-0.2837,  0.9451, -0.3490,  0.4564]), directed = False
relabel_nodes = False
    @withPackage('cudf')
    @withPackage('cugraph')
    @pytest.mark.parametrize('edge_weight', [None, torch.randn(4)])
    @pytest.mark.parametrize('directed', [True, False])
    @pytest.mark.parametrize('relabel_nodes', [True, False])
    def test_from_cugraph(edge_weight, directed, relabel_nodes):
        import cudf
        import cugraph
        from torch.utils.dlpack import to_dlpack
    
        if directed:
            edge_index = torch.tensor([[0, 1, 1, 2], [1, 0, 2, 1]])
        else:
            edge_index = torch.tensor([[0, 1], [1, 2]])
    
        if edge_weight is not None:
            edge_weight[:edge_index.size(1)]
    
        G = cugraph.Graph(directed=directed)
        df = cudf.from_dlpack(to_dlpack(edge_index.t()))
        if edge_weight is not None:
            df['2'] = cudf.from_dlpack(to_dlpack(edge_weight))
    
        G.from_cudf_edgelist(
            df,
            source=0,
            destination=1,
            edge_attr='2' if edge_weight is not None else None,
            renumber=relabel_nodes,
        )
    
        cu_edge_index, cu_edge_weight = from_cugraph(G)
    
>       assert cu_edge_index.tolist() == edge_index.tolist()
E       assert [[1, 0], [2, 1]] == [[0, 1], [1, 2]]
E         At index 0 diff: [1, 0] != [0, 1]
E         Use -v to get the full diff
test/utils/test_convert.py:469: AssertionError
___________________________ test_spmm_basic[device1] ___________________________
device = device(type='cuda', index=0)
    @withCUDA
    def test_spmm_basic(device):
        src = torch.randn(5, 4, device=device)
        other = torch.randn(4, 8, device=device)
    
        out1 = src @ other
        out2 = spmm(src.to_sparse(), other, reduce='sum')
        out3 = spmm(SparseTensor.from_dense(src), other, reduce='sum')
        assert out1.size() == (5, 8)
>       assert torch.allclose(out1, out2)
E       AssertionError: assert False
E        +  where False = <built-in method allclose of type object at 0x7f5bcc3dcbc0>(tensor([[ 2.0477, -1.1278, -0.3330,  1.9007,  3.0619,  1.4840,  2.0230, -2.3307],\n        [ 1.2360,  2.8585, -1.6878, ...8,  1.4510],\n        [ 0.8829, -2.6074,  0.4[767](https://gitlab-master.nvidia.com/dl/dgx/pyg/-/jobs/53984430#L767),  2.0739,  3.5401,  2.6623,  0.9645, -1.9539]],\n       device='cuda:0'), tensor([[ 2.0484, -1.1278, -0.3330,  1.9013,  3.0617,  1.4845,  2.0231, -2.3308],\n        [ 1.2361,  2.8589, -1.6877, ...8,  1.4507],\n        [ 0.8829, -2.6078,  0.4[768](https://gitlab-master.nvidia.com/dl/dgx/pyg/-/jobs/53984430#L768),  2.0739,  3.5405,  2.6624,  0.9641, -1.[954](https://gitlab-master.nvidia.com/dl/dgx/pyg/-/jobs/53984430#L954)3]],\n       device='cuda:0'))
E        +    where <built-in method allclose of type object at 0x7f5bcc3dcbc0> = torch.allclose
test/utils/test_spmm.py:19: AssertionError
=============================== warnings summary ===============================
../../../usr/local/lib/python3.8/dist-packages/torch_geometric-2.3.0-py3.8.egg/torch_geometric/graphgym/config.py:19
  /usr/local/lib/python3.8/dist-packages/torch_geometric-2.3.0-py3.8.egg/torch_geometric/graphgym/config.py:19: UserWarning: Could not define global config object. Please install 'yacs' via 'pip install yacs' in order to use GraphGym
    warnings.warn("Could not define global config object. Please install "
../../../usr/local/lib/python3.8/dist-packages/torch_geometric-2.3.0-py3.8.egg/torch_geometric/graphgym/imports.py:14
  /usr/local/lib/python3.8/dist-packages/torch_geometric-2.3.0-py3.8.egg/torch_geometric/graphgym/imports.py:14: UserWarning: Please install 'pytorch_lightning' via  'pip install pytorch_lightning' in order to use GraphGym
    warnings.warn("Please install 'pytorch_lightning' via  "
../../../usr/local/lib/python3.8/dist-packages/optuna/storages/_rdb/models.py:35
  /usr/local/lib/python3.8/dist-packages/optuna/storages/_rdb/models.py:35: MovedIn20Warning: The ``declarative_base()`` function is now available as sqlalchemy.orm.declarative_base(). (deprecated since: 2.0) (Background on SQLAlchemy 2.0 at: https://sqlalche.me/e/b8d9)
    BaseModel: Any = declarative_base()
test/nn/conv/test_lg_conv.py::test_lg_conv
  /usr/local/lib/python3.8/dist-packages/torch/nn/modules/module.py:1480: UserWarning: operator() profile_node %570 : int = prim::profile_ivalue(%565)
   does not have profile information (Triggered internally at /opt/pytorch/pytorch/torch/csrc/jit/codegen/cuda/graph_fuser.cpp:105.)
    return forward_call(*args, **kwargs)
test/nn/models/test_basic_gnn.py: 1 warning
test/nn/models/test_rev_gnn.py: 20 warnings
  /usr/local/lib/python3.8/dist-packages/torch/storage.py:315: UserWarning: TypedStorage is deprecated. It will be removed in the future and UntypedStorage will be the only storage class. This should only matter to you if you are using storages directly.
    warnings.warn(message, UserWarning)
-- Docs: https://docs.pytest.org/en/stable/warnings.html
---------- coverage: platform linux, python 3.8.10-final-0 -----------
Coverage XML written to file coverage.xml
=========================== short test summary info ============================
FAILED test/datasets/test_elliptic.py::test_elliptic_bitcoin_dataset - TypeEr...
FAILED test/utils/test_convert.py::test_to_cugraph[False-True-edge_weight1]
FAILED test/utils/test_convert.py::test_to_cugraph[False-False-edge_weight1]
FAILED test/utils/test_convert.py::test_from_cugraph[True-True-None] - assert...
FAILED test/utils/test_convert.py::test_from_cugraph[True-True-edge_weight1]
FAILED test/utils/test_convert.py::test_from_cugraph[True-False-None] - asser...
FAILED test/utils/test_convert.py::test_from_cugraph[True-False-edge_weight1]
FAILED test/utils/test_convert.py::test_from_cugraph[False-True-None] - asser...
FAILED test/utils/test_convert.py::test_from_cugraph[False-True-edge_weight1]
FAILED test/utils/test_convert.py::test_from_cugraph[False-False-None] - asse...
FAILED test/utils/test_convert.py::test_from_cugraph[False-False-edge_weight1]
FAILED test/utils/test_spmm.py::test_spmm_basic[device1] - AssertionError: as...
==== 12 failed, 3723 passed, 493 skipped, 25 warnings in 296.83s (0:04:56) =====
```

### Environment

updated to latest pyg and pyg-lib after the last issue was addressed

--- a/test/utils/test_convert.py
+++ b/test/utils/test_convert.py
@@ -10,6 +10,7 @@ from torch_geometric.utils import (
     from_networkx,
     from_scipy_sparse_matrix,
     from_trimesh,
+    sort_edge_index,
     subgraph,
     to_cugraph,
     to_networkit,
@@ -414,7 +415,7 @@ def test_to_cugraph(edge_weight, directed, relabel_nodes):
         edge_index = torch.tensor([[0, 1], [1, 2]])
 
     if edge_weight is not None:
-        edge_weight[:edge_index.size(1)]
+        edge_weight = edge_weight[:edge_index.size(1)]
 
     graph = to_cugraph(edge_index, edge_weight, relabel_nodes, directed)
     assert isinstance(graph, cugraph.Graph)
@@ -426,11 +427,18 @@ def test_to_cugraph(edge_weight, directed, relabel_nodes):
     edge_list = edge_list.sort_values(by=['src', 'dst'])
 
     cu_edge_index = edge_list[['src', 'dst']].to_pandas().values
-    assert edge_index.tolist() == cu_edge_index.T.tolist()
-
+    cu_edge_index = torch.from_numpy(cu_edge_index).t()
+    cu_edge_weight = None
     if edge_weight is not None:
         cu_edge_weight = edge_list['weights'].to_pandas().values
-        assert edge_weight.tolist() == cu_edge_weight.tolist()
+        cu_edge_weight = torch.from_numpy(cu_edge_weight)
+
+    cu_edge_index, cu_edge_weight = sort_edge_index(cu_edge_index,
+                                                    cu_edge_weight)
+
+    assert torch.equal(edge_index, cu_edge_index)
+    if edge_weight is not None:
+        assert torch.allclose(edge_weight, cu_edge_weight)
 
 
 @withPackage('cudf')
@@ -449,7 +457,7 @@ def test_from_cugraph(edge_weight, directed, relabel_nodes):
         edge_index = torch.tensor([[0, 1], [1, 2]])
 
     if edge_weight is not None:
-        edge_weight[:edge_index.size(1)]
+        edge_weight = edge_weight[:edge_index.size(1)]
 
     G = cugraph.Graph(directed=directed)
     df = cudf.from_dlpack(to_dlpack(edge_index.t()))
@@ -466,9 +474,11 @@ def test_from_cugraph(edge_weight, directed, relabel_nodes):
 
     cu_edge_index, cu_edge_weight = from_cugraph(G)
 
-    assert cu_edge_index.tolist() == edge_index.tolist()
+    cu_edge_index, cu_edge_weight = sort_edge_index(cu_edge_index,
+                                                    cu_edge_weight)
 
-    if edge_weight is None:
-        assert cu_edge_weight is None
+    assert torch.equal(edge_index, cu_edge_index)
+    if edge_weight is not None:
+        assert torch.allclose(edge_weight, cu_edge_weight)
     else:
-        assert cu_edge_weight.tolist() == edge_weight.tolist()
+        assert cu_edge_weight is None

